content/en/platform/corda/5.1/deploying-operating/deployment/deploying/manual-bootstrapping.md

```diff
@@ -47,6 +47,8 @@ The following is an example properties file for a Kafka cluster using {{< toolti
 3. Use the {{< tooltip >}}Corda CLI{{< /tooltip >}} to assist in the creation of the topics prior to Corda installation in one of two ways:
     * [Topic Creation by Direct Connection](#topic-creation-by-direct-connection)
     * [Topic Creation by Scripting](#topic-creation-by-scripting)
+   
+   For information about the Corda CLI `topic` command's arguments, see the [Corda CLI reference]({{< relref"../../../reference/corda-cli/topic.md">}}).
```

Comment:
Should we put this after showcasing the use of `topic` command so that we are aligned with other cases.